templates/azure/terraform/expand_property_method.erb

```diff
@@ -74,6 +74,7 @@ func expand<%= sdk_marshal.resource -%><%= descriptor.func_name -%>(input <%= go
 <%        elsif property.is_a?(Api::Type::Array) -%>
     results := make([]<%= sdk_marshal.package -%>.<%= sdk_marshal.sdktype.go_type_name -%>, 0)
     for _, v := range input {
+      v := v.(map[string]interface{})
```

Comment:
```suggestion
        v := item.(map[string]interface{})
```

Reply:
Should it be `item := item.(map[string]interface{})` ?

The generated .go code is like [this](https://github.com/terraform-providers/terraform-provider-azurerm/compare/master...houkms:houk-dev#diff-072322d5bd7284c81dca26fd0afae3f3R348-R349)

Reply:
No, it should be `v := item.(map[string]interface{})`